src/main/thrift/parquet.thrift

```diff
@@ -754,13 +821,14 @@ struct ColumnMetaData {
   /** Byte offset from beginning of file to Bloom filter data. **/
   14: optional i64 bloom_filter_offset;
 
-  /** Size of Bloom filter data including the serialized header, in bytes.
```

Comment:
Seems the comment has been lost after rebasing.

Reply:
added back.

Reply:
LGTM. Thanks!